setupTest.js

```diff
@@ -1,3 +1,4 @@
 'use strict';
 
+process.env.CHOKIDAR_USEPOLLING = 'true';
```

Comment:
This seems to fix it locally, we will see what happens with CI.

Reply:
Maybe we should use `true` instead `'true'`?